UI/expanded_ride_driver.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Ride my Way | Carpool | Ride sharing</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" 
+    href="static/main.css" />
+  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+</head>
+<body>
+<div class="page">
+  <header>
+    <h1>RideMyWay</h1>
+    <p>
+      A carpooling web application that provides drivers with the ability to 
+      create ride offers and passengers to join available ride offers.
+    </p>
+  </header>
+  <hr>
+  <nav>
+    <a href="#">Back to Rides</a> |
+    <a href="index.html">Sign out</a>
+    <a href="#">
+      <img src="static/avatar.png" alt="Default user avatar">
+    </a>
+  </nav>
+  <div class="row">
+    <div class="col-25">
+      <select name="rideStatus" id="rideStatus" onchange="">
+        <option value="openRide" selected>Status: Open</option>
+        <option value="closedRide">Status: Closed</option>
+      </select>
+    </div>
+  </div>
+  <hr>
+  <table>
+    <th colspan="2">Ride Details</th>
+    <tr>
+      <th>Date Scheduled</th>
+      <td>01-01-2018</td>
+    </tr>
+    <tr>
+      <th>Ride ID</th>
+      <td>#1</td>
+    </tr>
+    <tr>
+      <th>Driver</th>
+      <td>arthur</td>
+    </tr>
+    <tr>
+      <th>Departure Point</th>
+      <td>Thika Rd Mall</td>
+    </tr>
+    <tr>
+      <th>Departure Time</th>
+      <td>07:00AM</td>
+    </tr>
+    <tr>
+      <th>Final Destination</th>
+      <td>petergriffin@email.com</td>
```

Comment:
I assume your final destination is not an email address

Reply:
Thanks for that :+1: